src/solvers/flattening/boolbv_div.cpp

```diff
@@ -10,56 +10,58 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include <util/std_types.h>
 
-bvt boolbvt::convert_div(const div_exprt &expr)
+bvt boolbvt::convert_div(const div_exprt &div_expr)
```

Comment:
@peterschrammel I rather dislike using different names for arguments in declaration and definition because that gets quite confusing with Doxygen (these functions aren't documented at the moment of course, but they probably should be).